fusor-ember-cli/app/routes/review/progress/overview.js

```diff
@@ -3,12 +3,17 @@ import Ember from 'ember';
 export default Ember.Route.extend({
   model: function() {
     var deployment = this.modelFor('deployment');
-    return this.store.find('foreman-task', deployment.get('foreman_task_uuid'));
+    return Ember.RSVP.hash({
+        //foremanTask: this.store.find('foreman-task', deployment.get('foreman_task_uuid')),
+        openstackDeployment: this.store.find('openstack-deployment', 'overcloud'),
+        openstackPlan: this.store.find('deployment-plan', 'overcloud'),
+        openstackNodes: this.store.find('node'),
+    });
   },
 
   setupController: function(controller, model) {
     controller.set('model', model);
-    controller.startPolling();
+    //controller.startPolling();
```

Comment:
The polling only works for an RHCI deployment, which doesn't exist in the workflow on the develop branch.